src/zenml/integrations/seldon/seldon_client.py

```diff
@@ -93,6 +127,9 @@ class SeldonDeploymentPredictiveUnit(BaseModel):
     ] = SeldonDeploymentPredictiveUnitType.MODEL
     implementation: Optional[str]
     modelUri: Optional[str]
+    parameters: List[SeldonDeploymentPredictorParameter] = Field(
+        default_factory=list
+    )
```

Comment:
Is the `parametrs` required for every deployment? or can we make this:
```suggestion
    parameters: Optional[List[SeldonDeploymentPredictorParameter]]
```

Reply:
It is not required indeed, and can be optional. It cannot be null in the k8s manifest though, so I need to check how optional parameters are serialised.

Reply:
These were made optional now. As far as I can see from manual testing, it still behaves as expected